build.js

```diff
@@ -513,7 +513,19 @@ function dataToHtml(skeleton, rawBrowsers, tests, compiler) {
     else interpolateResults(t.res);
 
     var id = escapeTestName(t.name);
-    var name = t.spec ? ('<a href="' + t.spec + '">' + t.name + '</a>') : t.name;
+    var name = t.name;
+    if (t.spec) {
+        name = '<a href="' + t.spec + '">' + t.name + '</a>';
+        if (t.mdn) {
+            name += ' <a href="' + t.mdn + '"><img src="/mdn.png" width="15" height="13" /></a>&nbsp;';
+        }
+    }
+
+    if (!t.spec && t.mdn) {
+        name += ' <a href="' + t.mdn + '"><img src="/mdn.png" width="15" height="13" /></a>&nbsp;';
```

Comment:
Could the link html be stored in only one place, or perhaps better, a function that takes an href? (This one needs a11y too)